cpu/native/async_read.c

```diff
@@ -52,7 +52,10 @@ void native_async_read_cleanup(void) {
     unregister_interrupt(SIGIO);
 
     for (int i = 0; i < _next_index; i++) {
-        real_close(_fds[i].fd);
+        /* don't close stdin */
```

Comment:
Why is this necessary, it works also without this change?

Reply:
Otherwise stdin won't work after a `reboot`

Reply:
Hm, I tested to comment it out and it still works after reboot.

Reply:
That's odd, `tests/shell` would fail because `stdin` would not get re-opened.

Reply:
Ah, it just looks like the stdin still works after reboot, but the node doesn't react.